tracing/README.md

```diff
@@ -98,36 +104,39 @@ Let's consider the `log` crate's yak-shaving example:
 ```rust
 #[macro_use]
 extern crate tracing;
-use tracing::field;
+use tracing::Level;
 
+# #[derive(Debug)] pub struct Yak(String);
+# impl Yak { fn shave(&mut self, _: u32) {} }
+# fn find_a_razor() -> Result<u32, u32> { Ok(1) }
```

Comment:
It's a shame this can't be hidden. Maybe it would be better if it was just correctly formatted and had comments saying that these methods are fake?

Reply:
:+1: